libs/atqr/domain/src/lib/valueObjects/notification.ts

```diff
@@ -0,0 +1,114 @@
+import { ValueObject } from '../../utils/valueObject';
+import { Guid } from '@tokilabs/lang';
+import { NotificationLogEntry } from './notificationLogEntry.valueObject';
+import { MessageTemplateId } from '../services';
+
+enum NotificationCategory {
+  EuDuvidoProductUpdates = 'EuDuvidoProductUpdates',
+  EuDuvidoPromotions = 'EuDuvidoPromotions',
+  NewInvitations = 'NewInvitations',
+  ChallengeUpdates = 'ChallengeUpdates',
```

Comment:
These names were changed. see: https://github.com/tokilabs/atqr/blob/ATQR-79-Create-Notifications-Value-Objects/libs/atqr/domain/src/lib/domain-spec/concept-set/Notifications.disl#L35

Reply:
copied from Notification.disl : 
enum NotificationCategory
    ProductUpdates
    Promotions
    NewInvitations
    ChallengeUpdates

it is exactly like that . You just told me to delete EuDuvido parts .. I did and now both disl and notification.types have this same enum

Reply:
Sorry, my comment is on the wrong lines. Check this comment: https://github.com/tokilabs/atqr/pull/93/files/71494496a3bbb849ad6cfc433e93b1963e077519#r1089385130

You can resolve this one